demo/src/pages/childWindow/interfaces/ILaunchConfig.tsx

```diff
@@ -0,0 +1,11 @@
+interface ILaunchConfig {
```

Comment:
Could this file just be a definition file instead? e.g. `ILaunchConfig.d.ts` + `export interface ILaunchConfig`